0000-template.md

```diff
@@ -0,0 +1,101 @@
+# Title (Ex. 0000: RFC Topic)
+- Authors: [your name](you@github-email) -- email is optional
+- Status: [PROPOSED](/README.md#proposed)
+- Since: 2019-12-26 (date you submit your PR)
+- Status Note: (explanation of current status)  
+- Supersedes: (link to anything this RFC supersedes)
+- Start Date: 2018-12-26 (date you started working on this idea)
+- Tags: feature, protocol
+
+## Summary
+
+One paragraph explanation of the feature.
+
+## Motivation
+
+Why are we doing this? What use cases does it support? What is the expected
+outcome?
+
+## Tutorial
+
+Explain the proposal as if it were already implemented and you
+were teaching it to another Enarx contributor or user. That generally
+means:
+
+- Introducing new named concepts.
+- Explaining the feature largely in terms of examples.
+- Explaining how Enarx contributors and/or users should *think* about the
+feature, and how it should impact the way they would use Enarx.
+- If applicable, provide sample error messages, deprecation warnings, or
+migration guidance.
+
+Some RFCs may be more aimed at contributors (e.g. for
+consensus internals); others may be more aimed at users.
```

Comment:
Might be a typo here, should we add the word "on" in between "consensus" and "internals" -> "consensus on internals"

Reply:
That's from the original, but it doesn't sound great. Using your wording, thanks.